url_handlers_using_quarkus/src/main/java/org/acme/getting/started/macos_url_handler.java

```diff
@@ -0,0 +1,48 @@
+package org.acme.getting.started;
+import java.io.*;
+/*Adding Custom URL Handler to MacOS for ide:// */
+ public class macos_url_handler{
+     public static void main(String[] args) throws Exception{
+
+             Runtime run = Runtime.getRuntime();  
+            //The best possible I found is to construct a command which you want to execute  
+            //as a string and use that in exec. If the batch file takes command line arguments  
+            //the command can be constructed a array of strings and pass the array as input to  
+            //the exec method. The command can also be passed externally as input to the method.  
+
+            Process p = null;  
+            String cmd = "mv ide_handler.app ~/Applications";  //Moving to Applications
```

Comment:
why this move ? does it make a functional difference ?

Reply:
its just that we've to execute it to register the URL handler....so i did minimal